attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java

```diff
@@ -377,6 +394,7 @@ public boolean equals(Object o) {
         && Objects.equals(metadata, that.metadata)
         && Objects.equals(definition, that.definition)
         && Objects.equals(scopeString, that.scopeString)
+        && description.equals(that.description)
```

Comment:
nit: new addition at the end - can we move this condition after `&& Objects.equals(internal, that.internal)`